watson-developer-cloud/discovery-components
# fix: better tree shaking with @carbon/icons-react and carbon-react-co…

#### What do these changes do/fix?

discovery-react-components does not properly import icons from @carbon/icons-react and carbon-react-components ListBox to allow for tree shaking. This changes these imports to use the /es/ path which allows them to be tree shaken.

fixes #202 

#### How do you test/verify these changes?
Import discovery-react-components into a new create-react-app app and use webpack-bundle-analyzer to see the vendors.js bundle size.

#### Have you documented your changes (if necessary)?
No.

#### Are there any breaking changes included in this pull request?
I don't believe so.
